Frontend/src/boot/common.js

```diff
@@ -39,6 +39,46 @@ const common = {
     return new Blob(byteArrays, {
       type: contentType
     });
+  },
+  /**
+   * Converts enum text (e.g. UNDER_REVIEW) to a readable format
+   * (e.g. Under Review)
+   */
+  convertEnumTextToReadableString: function(enumText) {
+    const capitalize = s => {
+      if (typeof s !== "string") return "";
+      let lower = s.toLowerCase();
+      return lower.charAt(0).toUpperCase() + lower.slice(1);
+    };
+
+    let words = enumText.split("_");
+    let result = "";
+    words.forEach(word => {
+      result += capitalize(word) + " ";
+    });
+    return result.trim();
+  },
+  /**
+   * Converts a SQL-formatted date (e.g. 2020-01-01) to a readable format
+   * (e.g. January 1st, 2020)
+   */
+  convertSQLDateToReadableString: function(date) {
+    return moment(date)
+      .utc()
+      .format("MMMM Do, YYYY");
+  },
+  /**
+   * Converts a dollar value in cents (e.g. 2000) to a normal dollar
+   * representation (e.g. $20.00)
+   */
```

Comment:
Does this mean everything will be stored in cents. Does it also mean the user will input their cent earnings or will there be a textbox where $20.00 is entered and its converted to cents back and forth between fronend and backend